### DISCUSSION TITLE
Improvement for the helptext

### DISCUSSION THREAD

**Discussion by marioekaputta** [bahasa-ai/otomatis-guidance#24] [General]
If we can implement this in the help text for some input designer's action (as an example), I think we can explain it better to the maker.
![image](https://user-images.githubusercontent.com/71392755/129838221-1431f482-55fe-4022-884b-96df90304a5f.png)

**Comment by mgilangjanuar** [comment]
Did you mean to support the markdown styles in help label? 🤔

**Comment by marioekaputta** [comment]
Yes. Markdown styles

**Comment by jundialwan** [comment]
Well noted @marioekaputta 

We can support the MD but the label styling need customization I think.
Need to check first how to do the custom rendering in MD.

---

Example from Markdown styles already implemented on the last step of setup trigger/action, specifically on "Test Guideline" section. The label is not styled like a tag.

<img width="971" alt="Screen Shot 2021-08-25 at 12 50 11" src="https://user-images.githubusercontent.com/7735122/130733736-4a5fda65-715f-4c21-9b47-d960a17af194.png">

**Reply by nurulai** [reply]
kayanya bisa d tambahin d style siih kaa.. 
![image](https://user-images.githubusercontent.com/62420151/130962846-2530a800-b3ee-4f35-8150-0bd1165d97c0.png)

klo tak cek dri demonya jg kayanya.. yg aslinya cuma dpt font family nya
![image](https://user-images.githubusercontent.com/62420151/130962902-1c138e52-411f-42e4-be28-773e4efcb611.png)
